daemon/cmd/kube_proxy_replacement.go

```diff
@@ -109,7 +109,12 @@ func initKubeProxyReplacementOptions() (strict bool) {
 			log.Fatalf("Invalid value for --%s: %s", option.NodePortAcceleration, option.Config.NodePortAcceleration)
 		}
 
-		if !option.Config.NodePortBindProtection {
+		if option.Config.KubeProxyReplacement == option.KubeProxyReplacementProbe {
+			// We let kube-proxy do the less efficient bind-protection in
+			// this case to avoid the latter throwing (harmless) warnings
+			// to its log that bind request is rejected.
+			option.Config.NodePortBindProtection = false
```

Comment:
Do we want to log something (e.g., info message) when we overwrite this setting?

Reply:
I was thinking about it, definitely not a warn, but won't people just ignore and never read the info messages? How effective is it? Dunno..

Reply:
I was thinking that anyone who notices the option is not taking effect (because it was overwritten) will likely check the logs. Apart from the documentation (static & verbose), the logs are pretty much the only opportunity we have of explaining what's happening (at runtime) to users. I agree it shouldn't be a warning.

:+1: on `cilium status --verbose`

Reply:
Ok, I'll extend status in general on kpr a bit, but as separate PR then.